src/libcore/num/wrapping.rs

```diff
@@ -12,6 +12,7 @@ use super::Wrapping;
 
 use ops::*;
 
+#[allow(unused_macros)]
```

Comment:
There is a FIXME below to get the remaining impls uncommented, including invocations of the macro.